addons/ofxAndroid/src/ofxAndroidUtils.h

```diff
@@ -11,8 +11,38 @@
 #include "ofConstants.h"
 #include "ofEvent.h"
 
+
+
+enum ANDROID_PERMISSION{
```

Comment:
This should be ofxAndroidPermission since that's the standard we use also the individual enumerations should be OFX_ANDROID_PERMISSION_READ_CALENDAR...